script.js

```diff
@@ -0,0 +1,122 @@
+const hour = document.querySelector('.hour');
+const minute = document.querySelector('.minute');
+const second = document.querySelector('.second');
+
+const updateTime = () => {
+  const currentTime = new Date();
+
+  let currentHour = currentTime.getHours();
+  const currentMinute = currentTime.getMinutes();
+  const currentSecond = currentTime.getSeconds();
+
+  if (currentHour > 12) {
+    currentHour -= 12;
+  }
+  hour.textContent = currentHour.toString();
+  minute.textContent = currentMinute.toString().padStart(2, '0');
+  second.textContent = currentSecond.toString();
+};
+setInterval(updateTime, 1000);
+updateTime();
+
+const menuBtn = document.querySelector('.menu-btn');
+const reaction = document.querySelector('.menu-btn_options');
+const nav = document.querySelector('.nav');
+const menuNav = document.querySelector('.menu-nav');
+const navItems = document.querySelectorAll('.menu-nav__item');
+let showMenu = false;
+
+function toggleMenu() {
+  if (!showMenu) {
+    reaction.classList.add('open');
+    nav.classList.add('open');
+    menuNav.classList.add('open');
+    navItems.forEach((item) => item.classList.add('open'));
+
+    showMenu = true;
+  } else {
+    reaction.classList.remove('open');
+    nav.classList.remove('open');
+    menuNav.classList.remove('open');
+    navItems.forEach((item) => item.classList.remove('open'));
+
+    showMenu = false;
+  }
+}
```

Comment:
- Since ```toggleMenu``` takes no parameter explicitly we can even further clean our implementation and abstract it into own module, what do you think?